qi-doc/scribblings/principles.scrbl

```diff
@@ -21,7 +37,7 @@
 
  A @deftech{flow} is either made up of flows, or is a native (e.g. Racket) @seclink["lambda" #:doc '(lib "scribblings/guide/guide.scrbl")]{function}. Flows may be composed using a number of combinators that could yield either linear or nonlinear composite flows.
```

Comment:
Dominik pointed out that this definition identifies flows semantically with functions (and also a couple of paragraphs below). That may not be the case in general, especially if we allow users to indicate the backend.